vm-image-spec.yaml

```diff
@@ -15,6 +15,10 @@ commands:
     user: postgres
     sysvInitAction: respawn
     shell: '/usr/local/bin/pgbouncer /etc/pgbouncer.ini'
+  - name: local-proxy
+    user: postgres
```

Comment:
TODO: should this be the postgres user? local-proxy works a bit like pgbouncer in a sense, which also uses the postgres user.